lib/config/browserslistTargetHandler.js

```diff
@@ -275,6 +275,26 @@ const resolve = browsers => {
 			node: [12, 0]
 		}),
 
+		promise: rawChecker({
+			chrome: 32,
+			and_chr: 32,
+			edge: 12,
+			firefox: 29,
+			and_ff: 29,
+			// ie: Not supported,
+			opera: 19,
+			op_mob: 19,
+			safari: 8,
+			ios_saf: 8,
+			samsung: [2, 0],
+			android: [4, 4],
```

Comment:
It should be `[4, 4, 3]`, however the `rawChecker` only support two numbers at most. Not sure if it's necessary to support the patch version.